extensions/amp-tiktok/validator-amp-tiktok.protoascii

```diff
@@ -32,7 +32,7 @@ tags: {  # <amp-tiktok>
   attr_lists: "extended-amp-global"
   attrs: {
     name: "data-src";
-    value_regex: "\\d";
+    value_regex: "(.+/)?\\d+/?";
```

Comment:
It would seem that a `value_url` would be better for the case of a TikTok URL value, but given that the `amp-tiktok` component is only parsing the ID out of the URL, this doesn't seem required.